data.json

```
@@ -0,0 +1,899 @@
+{
+  "assessedAt": "2025-04-22T11:00:00Z",
+  "assessedBy": {
+    "id": "mailto:mlieberman85@gmail.com",
+    "comment": "Automated scan + manual review based on internal checklist #789"
+  },
+  "assessmentComment": "Full Self-Assessment for OpenSSF Security Baseline for WTF Frontend",
+  "controls": [
+    {
+      "id": "OSPS-AC-01.01",
+      "conclusion": "non-compliant",
+      "evidence": [
+        {
+          "humanEvidence": {
+            "name": "manual-inspection",
+            "humanDescription": "Examined AuthN MFA requirements for the GitHub Organization and found lack of enforcement. Changing the setting could be disruptive to the organization and will require further planning to mitigate."
+          }
+        }
+      ]
+    },
+    {
+      "id": "OSPS-AC-02.01",
+      "conclusion": "compliant",
+      "evidence": [
+        {
+          "ellaEvidence": {
+            "name": "automated-scan",
+            "ella": {
+              "uri": "",
+              "uriData": {}
+            }
+          }
+        }
```

Again, [I'd flatten this by one level](https://docs.google.com/document/d/16zwe3eNwExvnaXLDUrkGNineZzB0r8arKE3G-cvNU0E/edit?tab=t.0#heading=h.789n7qdnlkan).  I'm not sure that `automated scan` helps here, and it seems to mix the `assessedBy` human ID with the automation.  We may want to instead do three examples:

* A human assessment
* An automated partial assessment
* A merged assessment by a merging tool given the two base documents.

```suggestion
        {
          "ella": {
            "uri": "https://docs.github.com/en/get-started/learning-about-github/access-permissions-on-github",
          }
        }
```